packages/grafana-data/src/utils/logs.ts

```diff
@@ -8,7 +8,7 @@ import { ArrayVector } from '../vector/ArrayVector';
 // first a label from start of the string or first white space, then any word chars until "="
 // second either an empty quotes, or anything that starts with quote and ends with unescaped quote,
 // or any non whitespace chars that do not start with quote
-const LOGFMT_REGEXP = /(?:^|\s)([\w\(\)\[\]\{\}]+)=(""|(?:".*?[^\\]"|[^"\s]\S*))/;
+const LOGFMT_REGEXP = /(?:^|\s)([\w()\[\]{}]+)=(""|(?:".*?[^\\]"|[^"\s]\S*))/;
```

Comment:
hi @simPod , thanks for the PR! regarding this change to `LOGFMT_REGEXP`, this is only called from functions marked as `obsolete`, they will be removed in the future, so right now i don't think we want to make changes to this part of the code.

(i cannot comment on the remaining changes in the PR)

thanks!

Reply:
```suggestion
const LOGFMT_REGEXP = /(?:^|\s)([\w\(\)\[\]\{\}]+)=(""|(?:".*?[^\\]"|[^"\s]\S*))/;
```